.buildkite/Manifest-v1.11.toml

```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.11.4"
+julia_version = "1.11.5"
```

Comment:
Could we try to not change the things in the `Manifest` other than what is necessary for your PR. I.e., lets keep the julia version we use for testing to be `11.4`. Or if we choose to update, lets do it in a separate PR that just handles the dependencies.